Client/mods/deathmatch/logic/CNametags.h

```diff
@@ -37,6 +37,9 @@ class CNametags
     unsigned int GetDimension(void) { return m_usDimension; }
     void         SetDimension(unsigned short usDimension) { m_usDimension = usDimension; }
 
+    unsigned int GetInterior(void) { return m_usInterior; }
+    void         SetInterior(unsigned short usInterior) { m_usInterior = usInterior; }

```

Comment:
why this exists if you don't using this ?

Reply:
GetDimension and SetDimension are not used either, I thought I should add GetInterior and SetInterior in header too. I should delete this?